### DISCUSSION TITLE
Use ICON_AxisD for micro step down

### DISCUSSION THREAD

**Discussion by gothcha** [Jyers/Marlin#689] [Ideas]
As there is now an icon for axis down, this could be used for the micro step down menu items in place of the up one.

**Comment by tome9111991** [comment]
Compile firmware with last commit
Uncomment in config 

#define CREALITY_DWIN_EXTUI_CUSTOM_ICONS

Update display with
https://github.com/Jyers/Marlin/blob/Ender_3_V2_Extensible_UI/Display%20firmware/dwin-ico-tools/Mlevel-arrows/9.ICO

**Reply by gothcha** [reply]
Apologies, I checked and you are correct the latest commit is using this icon.

**Reply by gothcha** [reply]
I was looking at an earlier photo of my screen!!

**Comment by tome9111991** [comment]
You use release .bin firmware?

You have to compile by your self because
#define CREALITY_DWIN_EXTUI_CUSTOM_ICONS is not uncomment in release
